test/managers/course_repository_test.dart

```diff
@@ -1714,14 +1714,14 @@ void main() {
             programCode: '999',
             numberOfCredits: 3,
             title: 'Cours générique',
-            review: evaluation);
+            review: review);
 
         SignetsAPIClientMock.stubGetCourses(
             signetsApi as SignetsAPIClientMock, username,
             coursesToReturn: [courseFetched]);
         SignetsAPIClientMock.stubGetCourseReviews(
             signetsApi as SignetsAPIClientMock, username,
-            session: session, evaluationsToReturn: [evaluation]);
+            session: session, evaluationsToReturn: [review]);
```

Comment:
```suggestion
            session: session, reviewsToReturn: [review]);
```
Sneaky little one trying to escape 👀 
Is it easily fixable or does it need to be addressed in the package too?

Reply:
No don't worry those are only references in notre-dame directly, it's easily fixable. 😄 If you see anything else let me know! It's such a big refactor I'm sorry hahaha

Reply:
Actually I'll need to fix it from package too you're right! Didn't saw it was from this stub function. But it's not that long anyway!

Reply:
Nice to know it's not too hard to fix! It feels like you've already had your share of refactoring for today 😂